react/components/EmptyStateIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react'
+
+const EmptyStateIcon: FC = () => {
+  return (
+    <svg
+      width="60"
+      height="60"
+      viewBox="0 0 60 60"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M55 42H15C14.7563 41.9999 14.521 41.9109 14.3384 41.7496C14.1557 41.5883 14.0382 41.3658 14.008 41.124L9.234 4H2C1.46957 4 0.960859 3.78929 0.585786 3.41421C0.210714 3.03914 0 2.53043 0 2C0 1.46957 0.210714 0.960859 0.585786 0.585786C0.960859 0.210714 1.46957 0 2 0H11C11.4874 0.000137551 11.9579 0.178243 12.3233 0.500854C12.6886 0.823464 12.9236 1.26838 12.984 1.752L14.015 10H59C59.1424 10 59.2832 10.0305 59.413 10.0894C59.5427 10.1482 59.6583 10.2341 59.7522 10.3413C59.846 10.4485 59.9158 10.5745 59.957 10.7108C59.9982 10.8472 60.0098 10.9908 59.991 11.132L55.991 41.132C55.959 41.3722 55.8409 41.5926 55.6586 41.7522C55.4764 41.9119 55.2423 41.9999 55 42ZM17 60C20.866 60 24 56.866 24 53C24 49.134 20.866 46 17 46C13.134 46 10 49.134 10 53C10 56.866 13.134 60 17 60ZM53 60C56.866 60 60 56.866 60 53C60 49.134 56.866 46 53 46C49.134 46 46 49.134 46 53C46 56.866 49.134 60 53 60Z"
+        fill="#979899"
+      />
+    </svg>
+  )
+}
+
+export default React.memo(EmptyStateIcon)
```

Comment:
Is this really needed for a simple svg without props?

Reply:
Nice question! No, it is not!
Apparently, it might even hurt performance, according to https://dmitripavlutin.com/use-react-memo-wisely/

Reply:
oh react and its abstraction leaks <3